doc/source/myrocks/variables.rst

```diff
@@ -1354,6 +1373,17 @@ failed insertion attempt in INSERT ON DUPLICATE KEY UPDATE.
 
 Enables the rocksdb iterator upper bounds and lower bounds in read options.
 
+.. variable:: rocksdb_enable_pipelined_write
+
+    :version 8.0.25-15: Implemented
+    :cli: ``--rocksdb-enable-pipelined-write``
+    :dyn: No
+    :scope: Global
+    :vartype: Boolean
+    :default: ``OFF``
+
+DBOptions::enable_pipelined_write for RocksDB.
+
```

Comment:
My bad, I should have added this information in the ticket.

Might be positive adding some info about the option:

 If enable_pipelined_write is true, separate write thread queue is
 maintained for WAL write and memtable write. A write thread first enter WAL
 writer queue and then memtable writer queue. Pending thread on the WAL
 writer queue thus only have to wait for previous writers to finish their
 WAL writing but not the memtable writing. Enabling the feature may improve
 write throughput and reduce latency of the prepare phase of two-phase
commit.

Reply:
Added.